synthesizer/program/src/resources/credits.aleo

```diff
@@ -708,7 +674,7 @@ finalize set_validator_state:
     get committee[r0] into r2;
 
     // Construct the updated committee state.
-    cast r2.microcredits r1 into r3 as committee_state;
+    cast r1 r2.commission into r3 as committee_state;
     // Update the committee state for the specified validator.
     set r3 into committee[r0];
 
```

Comment:
Optional addition

[Idea from Solana allow withdrawal address to change](https://docs.solanalabs.com/cli/usage#solana-vote-authorize-withdrawer):
```
// Must be called by withdrawal address.
function set_withdrawal_address:
    input r0 as address.public; // Bonded address that will have a new withdrawal address set.
    input r1 as address.public; // New withdrawal address.

    // Do not allow updating withdrawal address to same as bonded address.
    assert.neq r0 r1;

    async set_withdrawal_address self.caller r0 r1 into r1;
    output r1 as credits.aleo/set_withdrawal_address.future;

finalize set_withdrawal_address:
    input r0 as address.public; // Current withdrawal address.
    input r1 as address.public; // Bonded address that will have a new withdrawal address set.
    input r2 as address.public; // New withdrawal address.

    // Validate current withdrawal address against caller. Fail if withdraw address does not exist.
    get withdraw[r1] into r3;
    assert.eq r0 r3;

    // Update to new withdrawal addrss.
    set r2 into withdraw[r1];
```

Reply:
I think we should probably not add this now to minimize the complexity of changes.